### PR TITLE
[doc] Update INSTALL to match reality.

### PR DESCRIPTION
**Pull request by ejgallego**
[c.f] https://coq.inria.fr/bugs/show_bug.cgi?id=4270

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -48,6 +45,12 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      incuding gtksourceview, see INSTALL.ide for more details
 
+   Opam (https://opam.ocaml.org/) is recommended to install ocaml and
+   the correspondign packages.
```
Comment (Zimmi48) [inline INSTALL]: typo: corresponding

Comment (ejgallego) [reply INSTALL]: fixed thanks.

File: INSTALL [hunk 2]
```diff
@@ -48,6 +45,12 @@ WHAT DO YOU NEED ?
    - for Coqide, the Lablgtk development files, and the GTK libraries
      incuding gtksourceview, see INSTALL.ide for more details
 
+   Opam (https://opam.ocaml.org/) is recommended to install ocaml and
+   the corresponding packages.
+
+   $ opam install ocamlfind camlp5 lablgtk-extras
+
+   should get you a reasonable Ocaml enviroment to compile Coq.
```
Comment (psteckler) [inline INSTALL]: The `lablgtk-extras` is for CoqIDE, right? So it's not strictly needed. Should that be mentioned?

Comment (ejgallego) [reply INSTALL]: It is mentioned just above.

Comment (psteckler) [reply INSTALL]: Yes, but it's not mentioned that building CoqIDE is optional, so installing `lablgtk-extras` is optional.

**Review by Zimmi48 — APPROVED** [review]

File: INSTALL (hunk 2 above)
Comment (maximedenes) [reply INSTALL]: @ejgallego can you make it more precise?

Comment (maximedenes) [reply INSTALL]: I think everybody is busy so I'll merge as it is. @psteckler feel free to open a new PR to make it even better.

Comment (herbelin) [inline INSTALL]: Probably not worth an official new PR, but a "n" is missing in "enviroment".

Comment (maximedenes) [reply INSTALL]: Ok, I'll push a fix directly.

**Pull request merged**